### ISSUE TITLE
geo纹理贴图怎么制作和区域范围一致的纹理图片呢

### ISSUE THREAD

**Issue by zhouzhili** [apache/echarts#18999]
### What problem does this feature solve?

geojson数据和纹理图片是怎么贴合的呢，制作了一个区域的行政区，上面叠加了影像纹理，但是范围一致不对。

我的示例是：https://codesandbox.io/s/basic-line-chart-forked-v7mnvq?file=/index.js

在issues里看到了一张纹理和数据的贴合图，想问下有没有可以直观查看geojson数据和纹理的位置关系的示例。

![image](https://github.com/apache/echarts/assets/14259932/329f2aa8-72a2-4119-bb72-af43bd17fbe9)

### What does the proposed API look like?

想了解geo数据和贴图的位置还有取值关系是怎么确定的，如何才能制作一个符合行政区域的贴图

**Comment by helgasoft** [comment]
Tough to do, but possible. You need to grab the texture with exact same size/zoom as the _geojson_ map.
Then eventually you could play the positioning with _width/height_ values in 
```<div id="main" style="width: 650px; height: 735px;">```
[Demo Code](https://codepen.io/helgasoft/pen/abPbOde) + please local-host the **texture file** attached here. Sorry, cannot (or don't know how to) upload to _codepen.io_.
Good news is, once texture and map are nicely superimposed, roaming works great.
### texture file:
![hk](https://github.com/apache/echarts/assets/13038071/77959031-d197-45f2-9e79-2118d05a5928)
### chart:
![image](https://github.com/apache/echarts/assets/13038071/02c211d9-4ecd-46b8-8060-13072be5584e)

**Comment by zhouzhili** [comment]
@helgasoft  Thank you for your response. I eventually found a solution. What I did was to first create an EChart and then place an image element underneath the chart. I adjusted the size and position of the image to align with the boundaries of the map in the EChart. When the map boundaries matched the image, I hid the EChart, captured a screenshot of the composite, and used that screenshot as a texture.

like this：
![image](https://github.com/apache/echarts/assets/14259932/86884570-c520-453d-b709-73ab11b9c46b)

My texture image looks like this.
![image](https://github.com/apache/echarts/assets/14259932/3bcb6cd8-7d1d-4bc1-85f0-2070e53f5484)

The final result.
![image](https://github.com/apache/echarts/assets/14259932/80b835c7-cfff-4bca-999e-30236fad08b9)
